docs/open-source-project/release-notes/tizen-4-0-m3.md

```diff
@@ -0,0 +1,258 @@
+# Tizen 4.0 Public M3 Release Notes
+
+Release Date: Aug. 31, 2018
+
+
+## System (Kernel and System Framework)
+
+### Fixes
+
+- Security improvement
+  - Systemd patch has been applied for CVE-2018-1049.
+
+
+## System (Base)
+
+### New and changed features
+
+- The following Open-source libraries have been upgraded:
+  - file (5.32)
+    - CVE-2018-10360
+  - sqlite (3.21.0)
+    - CVE-2018-8740
+  - libxml (2.9.7)
+    - CVE-2018-9251
+  - libxslt (1.1.31)
+  - procps-ng (3.3.15)
+    - CVE-2018-1126, CVE-2018-1125, CVE-2018-1124, CVE-2018-1123, CVE-2018-1122
+  - tzdata (2017c)
+  - unzip (6.10c23)
+    - CVE-2018-1000031, CVE-2018-1000032, CVE-2018-1000033
+  - vim (8.0.586)
+
+
+## Application Framework
+
+### New and changed features
+
+- TIDL – Tizen Interface Definition Language has been added (Mobile profile)
```

Comment:
Tizen Interface Definition Language (TIDL) has been added (Mobile profile)

Reply:
TIDL is the mainly used term, so how about changing like this including period.
TIDL (Tizen Interface Definition Language) has been added (Mobile profile).

Reply:
"TIDL (Tizen Interface Definition Language) has been added (Mobile profile)."
looks ok to me.